sound/soc/intel/boards/sdw_rt711_rt1308_rt715.c

```diff
@@ -34,6 +34,7 @@
 #define MAX_NO_PROPS 2
```

Comment:
A generic comment about this patch: there seems to be quite complicated logic how grouped codecs are handled with SDW. Should this code be put in some common implementation file, so it could be used by other machine drivers as well. It would most of the group-related logic is not related to 711/1308/715, but is rather generic logic needed by SDW systems.

Reply:
my anther idea is : this machine driver can be a generic one. we only need to add codec info in table to support new platform. Also we need to rename the machine driver. How about it ? @kv2019i @plbossart

Reply:
@RanderWang that sounds ok to me. Just like with HDA, as SDW is a discoverable bus, a generic machine driver should be able to handle majority of the cases. I'm ok to do the rename in a later patch.